iset.mm

```diff
@@ -61068,6 +61078,248 @@ all reals (greatest real number) for which all positive reals are greater.
+                    -. inf ( B , A , R ) R C ) ) $=
+      ( wcel cv wbr wn wral cinf wa wrex infglbti expdimp rexalim con2d expimpd
+      syl6 ) AIGMZDNIJOZPDHQZHGJRIJOZPAUGSZUJUIUKUJUHDHTZUIPAUGUJULABCDEFGHIJKL
+      UAUBUHDHUCUFUDUE $.
+  $}
+
+  ${
+    $d A u v y z $.  $d B u v y z $.  $d C u v y z $.  $d R u v y z $.
+    $d u v y z ph $.
+    infminti.ti $e |- ( ( ph /\ ( u e. A /\ v e. A ) ) ->
+        ( u = v <-> ( -. u R v /\ -. v R u ) ) ) $.
+    infminti.2 $e |- ( ph -> C e. A ) $.
+    infminti.3 $e |- ( ph -> C e. B ) $.
+    infminti.4 $e |- ( ( ph /\ y e. B ) -> -. y R C ) $.
+    $( The smallest element of a set is its infimum.  Note that the converse is
+       not true; the infimum might not be an element of the set considered.
+       (Contributed by Jim Kingdon, 18-Dec-2021.) $)
+    infminti $p |- ( ph -> inf ( B , A , R ) = C ) $=
+      ( vz wcel cv wbr wa wrex simprr breq1 rspcev syl2an2r eqinftid ) ABMCDEFG
+      HIJLAGFNBOZENZGUDHPZQUFMOZUDHPZMFRKAUEUFSUHUFMGFUGGUDHTUAUBUC $.
+  $}
+
+  ${
+    $d u v x y z A $.  $d u v x y z R $.  $d u v x y z B $.  $d u v x ph $.
+    infmoti.ti $e |- ( ( ph /\ ( u e. A /\ v e. A ) ) ->
+        ( u = v <-> ( -. u R v /\ -. v R u ) ) ) $.
+    $( Any class ` B ` has at most one infimum in ` A ` (where ` R ` is
+       interpreted as 'less than').  (Contributed by Jim Kingdon,
+       18-Dec-2021.) $)
+    infmoti $p |- ( ph -> E* x e. A ( A. y e. B -. y R x /\
+                  A. y e. A ( x R y -> E. z e. B z R y ) ) ) $=
+      ( cv wbr wn wral wrex wi wa wrmo vex brcnv supmoti notbii imbi12i anbi12i
+      ccnv cnvti ralbii rexbii rmobii sylib ) ABKZCKZIUEZLZMZCHNZULUKUMLZULDKZU
+      MLZDHOZPZCGNZQZBGRULUKILZMZCHNZUKULILZURULILZDHOZPZCGNZQZBGRABCDEFGHUMAEF
+      GIJUFUAVCVLBGUPVFVBVKUOVECHUNVDUKULIBSZCSZTUBUGVAVJCGUQVGUTVIULUKIVNVMTUS
+      VHDHULURIVNDSTUHUCUGUDUIUJ $.
+
+    infeuti.2 $e |- ( ph -> E. x e. A ( A. y e. B -. y R x /\
+                      A. y e. A ( x R y -> E. z e. B z R y ) ) ) $.
+    $( An infimum is unique.  (Contributed by Jim Kingdon, 19-Dec-2021.) $)
+    infeuti $p |- ( ph ->
+                    E! x e. A ( A. y e. B -. y R x /\
+                    A. y e. A ( x R y -> E. z e. B z R y ) ) ) $=
+      ( cv wbr wn wral wrex wi wa wrmo wreu infmoti reu5 sylanbrc ) ACLZBLZIMNC
+      HOUEUDIMDLUDIMDHPQCGORZBGPUFBGSUFBGTKABCDEFGHIJUAUFBGUBUC $.
+  $}
+
+  ${
+    $d A u v x $.  $d B u v x $.  $d R u v x $.  $d ph u v x $.
+    infsnti.ti $e |- ( ( ph /\ ( u e. A /\ v e. A ) ) ->
+      ( u = v <-> ( -. u R v /\ -. v R u ) ) ) $.
+    infsnti.b $e |- ( ph -> B e. A ) $.
+    $( The infimum of a singleton.  (Contributed by Jim Kingdon,
+       19-Dec-2021.) $)
+    infsnti $p |- ( ph -> inf ( { B } , A , R ) = B ) $=
+      ( csn cinf ccnv csup df-inf cnvti supsnti syl5eq ) AEIZDFJQDFKZLEQDFMABCD
+      ERABCDFGNHOP $.
+  $}
+
+  $( The infimum regarding an empty bas set is always the empty set.
+     (Contributed by AV, 4-Sep-2020.) $)
```

Comment:
typo bas -> base (I'll correct this in set.mm)

Reply:
I've put a fix for both set.mm and iset.mm into this pull request (if there is some reason why fixing it for set.mm here is a bad idea, I can change it, but seemed easier to do both at once).